lib/multivm/src/lib.rs

```diff
@@ -54,4 +75,19 @@ pub fn simulate_tx<S: ReadStorage, PS: PreimageSource>(
     }
 }
 
-pub const LATEST_EXECUTION_VERSION: u32 = 1;
+pub const LATEST_EXECUTION_VERSION: u32 = 2;
+
+pub fn proving_run_execution_version(forward_run_execution_version: u32) -> u32 {
```

Comment:
We are bringing proving-related stuff into multivm crate - this is a worrying sign

Reply:
Please see module description in `lib.rs`. This is a module providing unified interface for running blocks and simulating transactions. From my POV it was not meant to understand the nuances of proving. Just to be clear VKs and app.bins should only be stored/exposed for the last two versions. What we are doing right now seems to be that multivm holds both all historical forward systems AND also VKs/app.bins. This is a misdesign IMHO

Reply:
I removed apps::v1 and VKs from multivm. So now we only have `proving_run_execution_version` and `apps::v2` as a prover related code in multivm

Reply:
Thanks! FWIW I do think it's cleaner now